include/deal.II/lac/dynamic_sparsity_pattern.h

```diff
@@ -342,7 +342,9 @@ class DynamicSparsityPattern : public Subscriptor
                           const IndexSet &rowset = IndexSet());
 
   /**
-   * Create a square SparsityPattern using the index set.
+   * Create a square SparsityPattern using the given index set. The total size
+   * is given by the size of @p indexset and only rows contained in @p
+   * indexset are stored (the @p rowset).
```

Comment:
I'd just omit the parenthetical remark and just say "...and only rows corresponding to indices in @p indexset are stored on the current processor."